flixel/FlxObject.hx

```diff
@@ -1313,8 +1313,13 @@ class FlxObject extends FlxBasic
 		}
 
 		// fill static graphics object with square shape
-		gfx.lineStyle(1, color, 0.75);
-		gfx.drawRect(rect.x + 0.5, rect.y + 0.5, rect.width - 1.0, rect.height - 1.0);
+		gfx.beginFill(color, 0.5);
+		var size = Math.max(1 / zoom, 1);
+		
+		gfx.drawRect(rect.x - size, rect.y - size, size, rect.height + size);
+		gfx.drawRect(rect.x, rect.y - size, rect.width + size, size);
+		gfx.drawRect(rect.x + rect.width, rect.y, size, rect.height - size);
+		gfx.drawRect(rect.x, rect.y + rect.height - size, rect.width + size, size);
```

Comment:
Will these have stronger color in the overlapping parts? Rather than drawing 4 rects could we specify 8 points to make a hollow rect shape?
also 0.5 alpha should be .75

Reply:
Nope, this won't have stronger color cos they're not overlapping, they're drawn in a specific way so they don't touch, that's why all the weird math between the positions and the sizes lol

The alpha was set `0.5` before I changed it, so it was like that. altho I could change it to .75 no problem.